### ISSUE TITLE
Packages=libuv won't install

### ISSUE THREAD

**Issue by ki9us** [systemd/mkosi#1026]
I'm making mkosi build files for [`xmrig/xmrig`](https://github.com/xmrig/xmrig).  The image builds correctly but xmrig in the container throws this error:

```
/usr/local/bin/xmrig: error while loading shared libraries: libuv.so.1: cannot open shared object file: No such file or directory
```

This is unexpected because I've included `Packages=libuv` in the `mkosi.default`:

```
[Distribution]
Distribution=arch

[Output]
Format=directory
Output=/var/lib/machines/xmrig

[Packages]
BuildPackages=
        base-devel
        cmake
        hwloc
Packages=libuv

[Content]
Password=root
WithNetwork=yes
```

When I install `libuv` with pacman on my archlinux desktop, it appears to install these libraries:  

```sh
# ls -l /usr/lib/libuv*
lrwxrwxrwx 1 root root     14 Jan 13 15:16 /usr/lib/libuv.so -> libuv.so.1.0.0
lrwxrwxrwx 1 root root     14 Jan 13 15:16 /usr/lib/libuv.so.1 -> libuv.so.1.0.0
-rwxr-xr-x 1 root root 198640 Jan 13 15:16 /usr/lib/libuv.so.1.0.0
```

I added `ls -l /usr/lib/libuv*` to `mkosi.build` and it _does_ show those files in the build image.  (Must be a dependency for one of the `BuildPackages`)

But when I boot into the final image with systemd-nspawn, `ls -l /usr/lib/libuv*` returns nothing, which is probably why xmrig throws the error.  It's not in `/lib` or `/usr/local/lib` either.  

Not sure if this is a bug or I'm doing something wrong.  How do I debug if/where mkosi installs libuv?

**Comment by ki9us** [comment]
Here's the complete config: https://github.com/xmrig/xmrig/compare/master...ki9us:master

**Comment by behrmann** [comment]
Thanks for this pretty complete bug report! Could you also post mkosi's output to a gist or pastebin?

**Comment by ki9us** [comment]
I was having trouble making an image because my connection was poor, so I uploaded the source to my vps running debian bullseye.  I got an error about pacman not being installed and [switched the mkosi build to debian](https://github.com/ki9us/xmrig-mkosi/commit/331a5c1465347c9280a388082547cb3b96eae014) instead of installing pacman.  Also installed mkosi 13 from pip on the vps because 5 from apt didn't work.  

So here's the build in debian: https://gist.github.com/ki9us/6587f42731683aaa756a809847f0de64#file-mkosi-debian-log

Here's the build in arch, though there are some errors that might be connection trouble.  Might be helpful though since I got the same error about missing libuv.so.1 on both distros.  
https://gist.github.com/ki9us/6587f42731683aaa756a809847f0de64#file-mkosi-arch-log

I tried to `apt install hwloc libuv1` inside the debian container on my vps and then restart the sysd service and that worked.
